install/lenses.csv

```diff
@@ -0,0 +1,3 @@
+Make,Model,Serial,Mount,View,Attachment,Start Date,End Date
```

Comment:
Make: Mobotix AG
Model: M15-43D *Would need to list each lens type in other repo, could probably remove the D as this just states its as a day lens?*
Serial: 43D-15430
Mount: WHWI *connects to install/camera*
View: 01
Attachment: Day